packages/snap-preact-components/src/components/Organisms/Facet/Facet.tsx

```diff
@@ -242,6 +285,16 @@ export const Facet = observer((properties: FacetProps): JSX.Element => {
 							</div>
 						}
 					>
+						{searchable && searchableFacet.allowableTypes.includes(facet.display) && (
+							<div className="ss__facet--search">
+								<div className="ss__facet__search--wrapper">
+									<div className="ss__facet__search--icon">
+										<Icon icon="search" />
+									</div>
+									<input type="search" className="input" onChange={searchableFacet.searchFilter} placeholder={`Search ${facet.label}`} />
+								</div>
+							</div>
```

Comment:
From an implementations point of view, it would be hard to make changes to this HTML. Let' refactor this to a new `SearchInput` (molecule) component so that it can be customized with props (for changing placeholder text, search icon, close icon (also move away from using type='search' to be able to style the 'x'))
When creating a new component, you'll also need to create the SearchInput.stories.txs storybook file and readme.md for it's documentation